content/docs/get-started/gcp/destroy-stack.md

```diff
@@ -12,46 +12,60 @@ menu:
 aliases: ["/docs/quickstart/gcp/destroy-stack/"]
 ---
 
-Now that we've seen how to deploy changes to our program, let's clean up and tear down the resources that are part of our stack.
+Now that you've seen how to deploy changes to our program, let's clean up and tear down the resources that are part of your stack.
 
 To destroy resources, run the following:
 
 ```bash
 $ pulumi destroy
 ```
 
-You'll be prompted to make sure you really want to delete these resources.
+You'll be prompted to make sure you really want to delete these resources. This can take a minute or two; Pulumi waits until all resources are shut down and deleted before it considers the destroy operation to be complete.
```

Comment:
```suggestion
You'll be prompted to make sure you really want to delete these resources. This can take a minute or two; Pulumi waits until all resources are shut down and deleted before it considers the destroy operation complete.
```